### PR TITLE
Add smart app banner

### LINKED ISSUE #6: Add App Store app header
... so that it shows on the top when you load the page on iOS devices - probably some kind of meta attribute or so.

### PR DESCRIPTION
**Pull request by trabaldini**
closes #6

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
 		<meta name="language" content="de"/>
 		<meta name="content-language" content="de"/>
 		<meta name="publisher" content="simplabs, Malthe Luda" />
+    <meta name="apple-itunes-app" content="app-id=918237891, affiliate-data=myAffiliateData, app-argument=myURL">
```
Comment (marcoow) [inline _layouts/default.html]: Can we leave out `affiliate-data` and `app-argument` as we only provide dummy data anyway?

**Review by marcoow — COMMENTED** [review]
nice!

File: _layouts/default.html (hunk 1 above)
Comment (trabaldini) [reply _layouts/default.html]: Yes, the `affiliate-data` and `app-argument` are optional
https://developer.apple.com/library/content/documentation/AppleApplications/Reference/SafariWebContent/PromotingAppswithAppBanners/PromotingAppswithAppBanners.html

**Review by marcoow — APPROVED** [review]

**Comment by marcoow** [comment]
👍 This should be done for the .nl, .it, .fr, .com websites as well

**Pull request merged**